docs/reference.md

```diff
@@ -115,6 +116,34 @@ Application authors may want to use the [rhasspy-client](https://pypi.org/projec
 * `/api/events/log`
     * Listen for log messages published as plain text
 
+## MQTT API
+
+Rhasspy implements part of the [Hermes](https://docs.snips.ai/reference/hermes) protocol. Various services of Rhasspy can be configured to pass along MQTT messages or to react to MQTT messages following the Hermes protocol.
+
+* `hermes/audioServer/<SITE_ID>/playBytes/<REQUEST_ID>`
+    * Rhasspy publishes 16 kHz, 16-bit mono audio in WAV format to this topic.
```

Comment:
This is not entirly correct, if you set TTS to Google Wavenet you can set a different kHz. 
Also, you can select diffent feedback sounds with different audio format, which is played fine by Rhasspy

Reply:
Thanks for having a look at this! So, the [audio output](https://rhasspy.readthedocs.io/en/latest/audio-output/#mqtthermes) documentation is also not entirely correct then:

> Rhasspy will always try to send 16 kHz, 16-bit mono audio.

Right?

Reply:
I have changed the wording somewhat now, also in the audio output documentation.

Reply:
I think indeed the audio output is also not completely correct.